config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -33,6 +33,54 @@ presets:
     mountPath: /etc/azure-ssh
     readOnly: true
 
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-aks-engine-azure-windows
+    always_run: false
+    optional: true
+    trigger: (?m)^/test( | .* )pull-kubernetes-e2e-aks-engine-azure-windows,?($|\s.*)
+    rerun_command: /test pull-kubernetes-e2e-aks-engine-azure-windows
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-acsengine: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--repo=k8s.io/kubernetes=$(PULL_REFS)"
+        - "--upload=gs://kubernetes-jenkins/pr-logs/"
+        - "--timeout=480"
+        - "--scenario=kubernetes_e2e"
+        - --
+        - "--test=true"
```

Comment:
as per the CI error:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/12010/pull-test-infra-bazel/1112774511742160896

the job needs such an arg:
`- --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-aks-engine-azure-windows`

Reply:
Indeed. Added that.